lib/routes/subswap/swap/widgets/deposit_widget.dart

```diff
@@ -122,8 +122,6 @@ class _DepositWidgetState extends State<DepositWidget> {
       return texts.invoice_btc_address_warning_without_min_fee_account_connected(
         minSats,
         maxSats,
-        setUpFee,
```

Comment:
@ubbabeck If you revert the changes as suggested by @erdemyerebasmaz here https://github.com/breez/breezmobile/pull/1181#pullrequestreview-1558795997
Please update here as well, it is a good example of what I said here: https://github.com/breez/breezmobile/pull/1181#issuecomment-1662156491

Reply:
Reverted as it was reverted on breez translations